tests/scripts/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env bash
+# deploy.sh
+# Sets up the environment for the admission controller webhook in the active cluster.
+set -eo pipefail
+
+sudo cp /etc/kubernetes/admin.conf $HOME/
```

Comment:
If you change the BASE_DIR according to the other comment, could this script just call a symlink to the other one to avoid the duplication of the rest of the script after line 8?

Reply:
changed BASE_DIR to point to current dir... Have added a wrapper script